admin/app/components/organizations/information-section.js

```diff
@@ -146,10 +146,10 @@ export default class OrganizationInformationSection extends Component {
       return;
     }
 
-    this.args.organization.set('name', this.form.name.trim());
-    this.args.organization.set('externalId', !this.form.externalId ? null : this.form.externalId.trim());
-    this.args.organization.set('provinceCode', !this.form.provinceCode ? null : this.form.provinceCode.trim());
-    this.args.organization.set('email', !this.form.email ? null : this.form.email.trim());
+    this.args.organization.set('name', this.form.name);
+    this.args.organization.set('externalId', this.form.externalId);
+    this.args.organization.set('provinceCode', this.form.provinceCode);
+    this.args.organization.set('email', this.form.email);
     this.args.organization.set('credit', !this.form.credit ? null : this.form.credit);
```

Comment:
**question** : on change pas pour les crédits ?

Reply:
les crédits sont de type number donc ça ne fonctionne pas avec les transforms qu'on a créé.
opt 1 : inclure le type number dans le nullable-string
opt 2 : créer un transform `nullable-number` ( je pense qu'il faudrait changer le nom) spécialement pour le type number

Reply:
du coup on enlève pas le trim ? this.args.organization.set('credit', this.form.credit);

Reply:
ah si je l'ai enlevé mais pas commit 😅 je fixe ça !